From 770da6dd522f30c742fefe43948a84ede543c542 Mon Sep 17 00:00:00 2001
From: Dan Voiculeasa <dan.voiculeasa@windriver.com>
Date: Thu, 2 Jun 2022 20:20:50 +0300
Subject: [PATCH 2/2] Allow external bash completion

Don't generate the bash completion at build time, as it is different
than the one generated at runtime. Allow bash completion to be
generated at /etc/bash_completion.d .

Signed-off-by: Dan Voiculeasa <dan.voiculeasa@windriver.com>
---
 debian/python3-openstackclient.dirs | 1 +
 debian/rules                        | 3 ---
 2 files changed, 1 insertion(+), 3 deletions(-)
 create mode 100644 debian/python3-openstackclient.dirs

diff --git a/debian/python3-openstackclient.dirs b/debian/python3-openstackclient.dirs
new file mode 100644
index 0000000..bc3b680
--- /dev/null
+++ b/debian/python3-openstackclient.dirs
@@ -0,0 +1 @@
+etc/bash_completion.d/
diff --git a/debian/rules b/debian/rules
index f032e33..638ee8d 100755
--- a/debian/rules
+++ b/debian/rules
@@ -16,9 +16,6 @@ override_dh_auto_build:
 override_dh_auto_install:
 	pkgos-dh_auto_install --no-py2 --wheel
 
-	# Generate bash completion
-	pkgos-gen-completion --py3 openstack
-
 override_dh_auto_test:
 ifeq (,$(findstring nocheck, $(DEB_BUILD_OPTIONS)))
 	# See https://bugs.debian.org/1058097
-- 
2.34.1

